ra_aid/__main__.py

```
@@ -788,6 +806,130 @@ def build_status():
+            config_repo.set("mcp_use_enabled", False)
+            config_repo.set("active_mcp_servers", [])
+    
+    # Validate message is provided
+    if not args.message:
+        error_message = "No task provided"
+        print_error(error_message)
+        return
+    
+    # Store the human input in the database
+    human_input_repo = get_human_input_repository()
+    human_input_id = human_input_repo.create(args.message)
```

Ruff (0.8.2)

Local variable `human_input_id` is assigned to but never used

Remove assignment to unused variable `human_input_id`

(F841)